Dockerfile

```
@@ -11,6 +11,8 @@ ARG BUILD_TIMESTAMP
 ENV BUILD_TIMESTAMP=$BUILD_TIMESTAMP
 ARG GIT_COMMIT
 ENV GIT_COMMIT=$GIT_COMMIT
+ARG NEXT_PUBLIC_BLOBSCAN_RELEASE
+ENV NEXT_PUBLIC_BLOBSCAN_RELEASE=$NEXT_BLOBSCAN_RELEASE
```

Shouldn't this be `NEXT_PUBLIC_BLOBSCAN_RELEASE` instead of `NEXT_BLOBSCAN_RELEASE`?
```suggestion
ENV NEXT_PUBLIC_BLOBSCAN_RELEASE=$NEXT_PUBLIC_BLOBSCAN_RELEASE
```